recipes/minizip-ng/all/conanfile.py

```diff
@@ -83,7 +83,7 @@ def requirements(self):
         if self.options.with_bzip2:
             self.requires("bzip2/1.0.8")
         if self.options.with_lzma:
-            self.requires("xz_utils/5.4.5")
+            self.requires("xz_utils/5.6.0")
```

Comment:
```suggestion
            self.requires("xz_utils/5.4.5")
```

Let's revert unless there's a good reason to bump :)